### PR TITLE
[SOL] Add README.md

### PR DESCRIPTION
**Pull request by adam-maj**

### CODE REVIEW

File: packages/sdk/src/solana/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+Install the latest version of the SDK with either `npm` or `yarn`:
+
+```shell
+npm install @thirdweb-dev/solana
+```
+
+```shell
+yarn add @thirdweb-dev/solana
+```
```
Comment (jnsdls) [inline packages/sdk/src/solana/README.md:24]: this is not correct anymore

File: packages/sdk/src/solana/README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+## Quick Start
+
+The first thing to do to get started with Solana using thirdweb is to deploy a program. You can do this via the [Dashboard](https://thirdweb.com/dashboard), or via the SDK with the following snippet:
+
+```jsx
+import { ThirdwebSDK } from "@thirdweb-dev/solana";
```
Comment (jnsdls) [inline packages/sdk/src/solana/README.md:31]: nope

File: packages/sdk/src/solana/README.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  name: "New NFT",
+});
+const nft = await program.get(mintAddress);
+```
+
+## Learn More
+
+You can learn more about thirdweb and the Solana SDK with the following resources:
+
+- [Solana Docs](https://portal.thirdweb.com/solana)
```
Comment (jnsdls) [inline packages/sdk/src/solana/README.md:67]: is this still correct?

**Review by jnsdls — CHANGES_REQUESTED** [review]
readme needs fixes, also not a fan of putting readmes in the src (I understand why ur doing it but I feel like there's better solutions here)

File: packages/sdk/src/solana/README.md (hunk 1 above)
Comment (adam-maj) [reply packages/sdk/src/solana/README.md:24]: yea we just copy pasted from old version good catch

Comment (adam-maj) [reply packages/sdk/src/solana/README.md:24]: do yk why build is broken

Comment (jnsdls) [reply packages/sdk/src/solana/README.md:24]: no but looking at it

**Review by jnsdls — APPROVED** [review]
need to find a better way to have this readme but this is fine for now

**Comment by jnsdls** [comment]
@adam-maj the side-effect of this is that test caches break every-time u update the readme now lol, seems pretty bad. why can't we have 3 readmes top-level? 1 general sdk entry point 1 for evm 1 for sol specific instead?

**Comment by adam-maj** [comment]
> @adam-maj the side-effect of this is that test caches break every-time u update the readme now lol, seems pretty bad. why can't we have 3 readmes top-level? 1 general sdk entry point 1 for evm 1 for sol specific instead?

This is way better - I also didn't want to put it in src, we're just working in gather and wanted to fix it since docs are merged. Will change it rn on this branch

**Comment by adam-maj** [comment]
@jnsdls build still breaking

**Pull request merged**